Sources/Utilities/Storage/Session/PersistentSessionManager.swift

```diff
@@ -78,7 +78,11 @@ final class PersistentSessionManager: SessionManager {
     var isPersistentSessionIDMissing: Bool {
         persistentID == nil && isReturningUser
     }
-
+    
+    var hasNotRemovedLocalAuth: Bool {
```

Comment:
I think we can make this `private` now too 🙂

Reply:
It doesn't let me.. I think because of the protocol?

Reply:
Do we need it in the protocol still? I think it's not needed there either since we removed it from the coordinator 🙂